src/compass/s1_geocode_slc.py

```diff
@@ -62,6 +63,11 @@ def run(cfg):
     native_doppler = burst.doppler.lut2d
     orbit = burst.orbit
 
+    # Get range and azimuth LUTs
+    rg_lut, az_lut = compute_lut(burst,
```

Comment:
```suggestion
    rg_correction_lut, az_correction_lut = compute_lut(burst,
```
Verbosity for clarity.